interop/src/build/web/wasm.rs

```diff
@@ -19,7 +19,7 @@ use crate::TEST_SERVER_PORT;
 use color_eyre::eyre::Result;
 use std::net::SocketAddr;
 
-pub async fn find_wasm_file(deploy_path: &std::path::Path) -> Result<std::path::PathBuf> {
+pub(crate) async fn find_wasm_file(deploy_path: &std::path::Path) -> Result<std::path::PathBuf> {
```

Comment:
Why does this need to be `pub(crate)`? I don't see it used outside of this module.